### DISCUSSION TITLE
Adding viewflow to existing django application

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [viewflow/viewflow#404] [General]
I ran the pip install django-viewflow and migrate and suspect I need to modify the urls for viewflow since my existing application's urls are already set and working.  However I didn't find the urls.py under viewflow so unsure how to finish the installation and start working with viewflow.  

Is there documentation on adding viewflow to existing django applications?

**Comment by kmmbvnr** [comment]
You can just add required viewsts into the urls configurations

https://docs.viewflow.io/overview/quick_start.html#initial-site-structure

**Reply by michaeloliveraz** [reply]
Well, thanks, but I am not talking about quickstart; that is fine. I am talking about adding viewflow to an existing Django application.  I have the venv and have it activated, done the **pip install django-viewflow** and modified my site settings to add viewflow.  I do not see viewflow.urls to add to my urlpatterns.

**Reply by michaeloliveraz** [reply]
Also, I do see process lists and tasks in the Admin page under workflow, so I must assume django-viewflow is installed and available.  So it appears just figuring out how to add viewflow urls to the site.urls.py is the next step before I can started adding my own flows.

**Comment by michaeloliveraz** [comment]
What about getting the full repo of the online viewflow demo.  Working from that perhaps I can merge that demo with my app and have a solid starting point.

**Reply by michaeloliveraz** [reply]
I am assuming you have a repo for the online viewflow demo.  That would be a very good way to see how you do things, which should be public.

**Comment by kmmbvnr** [comment]
Since Viewflow 2.0 all integration happens through pre-defined class-based url configurations - Viewsets https://docs.viewflow.io/crud/viewset.html

So depends on your task you just need to a Viewset, that could be CRUD, Workflow, Dashboard Viewset into your urls.py

As for code demo, there is a viewflow cookbook that could be used as a reference - https://github.com/viewflow/cookbook

But generally, my recommendation  - just work on your task. If you need to check demo like it''s available at https://demo.viewflow.io/intro/
